### ISSUE TITLE
Worked on new 3ds. Force closed music app on O3ds

### ISSUE THREAD

**Issue by ehnobigdeal** [nedwill/soundhax#73]
Just went through the whole process on my N3DS and everything worked fine. Said what the hell lets do the old too. Downloaded new files pertaining to the O3DS and did the same steps as on the New. get a red and green screen on the music app before the system force closes it. Just wanted to report on it and to be able to get updated when there is a fix. Version 11.2.0-35U Old (usa) Thankyou for all the work you do though! it is amazing!

**Comment by yellows8** [comment]
Make sure you used the Old3DS m4a.

**Comment by ehnobigdeal** [comment]
yes i did. i tried the dl from here for the o3ds m4a US as well as the one for the o3ds US downloaded from soundhax.com

**Comment by yellows8** [comment]
"/otherapp.bin" is missing from your SD.

**Comment by ehnobigdeal** [comment]
no its not. downloaded and renamed it. but its just otherapp.bin not /otherapp.bin? do you need the /?

**Comment by yellows8** [comment]
The "/" refers to the filepath not filename .......

**Comment by ehnobigdeal** [comment]
ah my bad didnt catch that xD but yes it is definitely on there. like i said i did everything that i did for the new 3ds (which worked perfectly) that i did for the old 3ds except with the differing files.
